recipes/seurat-scripts/meta.yaml

```diff
@@ -21,6 +20,7 @@ requirements:
         - r-seurat 2.3.1
         - r-optparse
         - r-workflowscriptscommon
+        - mscorefonts
```

Comment:
This will hopefully soon be taken care of by https://github.com/conda-forge/r-base-feedstock/pull/104